docs/content/service/other/apache-solr.md

```diff
@@ -83,6 +83,12 @@ Legend:
 
 - `<image-repo>:<software-version>[-<image-stability-tag>][-<flavor>]`
 
+To use one of these versions put this variable in your .docksal/docksal.env file. For example to use the latest version put this in your docksal.env file
```

Comment:
We use `fin config set` is the preferred way for settings a project configuration variable. Same result, but faster.

Take a look at https://docs.docksal.io/service/db/settings/#mysql-versions.